linux_os/guide/services/http/securing_httpd/httpd_configure_log_format/rule.yml

```diff
@@ -24,7 +24,7 @@ identifiers:
     cce@rhel7: "80548-1"
 
 references:
-    stigid: WA00612
```

Comment:
@matejak Why is it twice in `ssg-rhel7-ds.xml`?

See:
```
<reference href="https://public.cyber.mil/stigs/downloads/?_dl_facet_stigs=operating-systems%2Cunix-linux">WA00612</reference>
<reference href="https://public.cyber.mil/stigs/downloads/?_dl_facet_stigs=operating-systems%2Cunix-linux">RHEL-07-WA00612</reference>
```

Reply:
@matejak Actually, a lot if items are duplicated, once with the RHEL-07- prefix, and once without that prefix. 

Try this:
```
grep "https://public.cyber.mil/stigs/downloads/?_dl_facet_stigs=operating-systems%2Cunix-linux" build/ssg-rhel7-ds.xml
```

Is it intentional?

Reply:
Thanks for noticing, the duplication is a bug - there should be only IDs with prefixes.